packages/queries/src/iam/delegation.ts

```diff
@@ -0,0 +1,236 @@
+import {
+  generateChildAccountToken,
+  getChildAccountProfile,
+  getDefaultDelegationAccess,
+  listChildAccountDelegates,
+  listChildAccounts,
+  listDelegatedChildAccountsForUser,
+  listMyDelegatedChildAccountProfiles,
+  updateChildAccountDelegates,
+  updateDefaultDelegationAccess,
+} from '@linode/api-v4';
+import { createQueryKeys } from '@lukemorales/query-key-factory';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+
+import type {
+  APIError,
+  IamUserRoles,
+  Params,
+  ResourcePage,
+  Token,
+} from '@linode/api-v4';
+
+export const delegationQueries = createQueryKeys('delegation', {
+  lists: {
+    contextQueries: {
+      childAccounts: ({ params }: { params: Params }) => ({
+        queryFn: () => listChildAccounts({ params }),
+        queryKey: [params],
+      }),
+      delegatedChildAccountsForUser: ({
+        username,
+        params,
+      }: {
+        params: Params;
+        username: string;
+      }) => ({
+        queryFn: () => listDelegatedChildAccountsForUser({ username, params }),
+        queryKey: [username, params],
+      }),
+      childAccountDelegates: ({
+        euuid,
+        params,
+      }: {
+        euuid: string;
+        params: Params;
+      }) => ({
+        queryFn: () => listChildAccountDelegates({ euuid, params }),
+        queryKey: [euuid, params],
+      }),
+      myDelegatedChildAccountProfiles: ({ params }: { params: Params }) => ({
+        queryFn: () => listMyDelegatedChildAccountProfiles({ params }),
+        queryKey: [params],
+      }),
+      childAccountProfile: ({ euuid }: { euuid: string }) => ({
+        queryFn: () => getChildAccountProfile({ euuid }),
+        queryKey: [euuid],
+      }),
+      defaultAccess: () => ({
+        queryFn: () => getDefaultDelegationAccess(),
+        queryKey: ['defaultAccess'],
```

Comment:
because no params are used for this query, we don't need a queryKey. 

Currently this will build a query key that looks like `["delegation", "lists", "defaultAccess", "defaultAccess"]`

The extra `defaultAccess` isn't nessary

```suggestion
        queryKey: null,
```